globus_automate_client/flows_client.py

```diff
@@ -367,7 +409,13 @@ def flow_action_status(
         :param flow_action_id: The ID specifying the Action for which's status
             we want to query
         """
-        ac = self._action_client_for_flow(flow_id, flow_scope)
+        if "authorizer" in kwargs:
```

Comment:
maybe an internal helper for this to avoid duplicating the code. In said helper, can treat kwargs simply as a dict.